apps/test/unit/lib/kits/maker/commandsTest.js

```diff
@@ -42,61 +42,61 @@ describe('maker/commands.js', () => {
     it('display warning when reserved pin 1 is used', () => {
```

Comment:
It might be good to add a test case here to verify that `outputError` is called in the case of an invalid pin ID. This could also be a follow-up though if you'd prefer to ship this first.

Reply:
Thanks - added a unit test for invalid pin id.